lib/core/Canvas.js

```diff
@@ -427,11 +542,16 @@ Canvas.prototype.toggleMarker = function(element, marker) {
 };
 
 Canvas.prototype.getRootElement = function() {
-  if (!this._rootElement) {
-    this.setRootElement({ id: '__implicitroot', children: [] });
+  var plane = this.getActivePlane();
+  if (!plane.rootElement) {
```

Comment:
Proposal: Why not move this logic into `getActivePlane`.

It feels odd that a default plane is always provided but needs to be monkey patched to inject the root element.

Cleaner if possible:

* Upon accessing the active plane, create it + the root element.
* Provide an alternative API to create a new plane from